src/main/plugin-metadata/plugin-security.policy

```diff
@@ -4,4 +4,5 @@ grant {
     permission java.lang.RuntimePermission "accessDeclaredMembers";
     permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
     permission java.lang.RuntimePermission "setContextClassLoader";
+    permission java.net.NetPermission "accessUnixDomainSocket";
```

Comment:
@martin-gaievski can we try removing this from the PR? This permission is already granted by the test-framework. See https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/resources/org/opensearch/bootstrap/test-framework.policy#L188

Last night, the issue was that https://github.com/opensearch-project/OpenSearch/pull/18764 was still unmerged. Now that its merged the tests are passing and this entry shouldn't be necessary.